### ISSUE TITLE
Full bright (Numlock 2) crashes dreamseeker

### ISSUE THREAD

**Issue by jinxynii** [SijyKijy/SpaceStation13-Cheat#2]
-

**Comment by SijyKijy** [comment]
It was written for version 513.
If i'm not mistaken, BYOND Control is dead. Is there link?

**Comment by SijyKijy** [comment]
Do you have a way to send them to me? :)

**Comment by SijyKijy** [comment]
If anything, I've already started the process upgrading to version 515. See branch [515_upd](https://github.com/SijyKijy/SpaceStation13-Hack/tree/515_upd). But fullbright isn't working yet

**Comment by jinxynii** [comment]
https://easyupload.io/ij6pqy This is what I have. Replace the main.dll in bin with the one you want to use-- though I'm almost 100% certain the 515 one never worked. Though, if you use BYOND 514, and join a 515 server, sometimes you can still join, and if it does, the whole program still works.

**Comment by jinxynii** [comment]
Let me know if this one doesn't work, I have another one that I've been using more recently. I just don't know which one's which. :P

**Comment by SijyKijy** [comment]
> Let me know if this one doesn't work, I have another one that I've been using more recently. I just don't know which one's which. :P

Send them both in at once. I'll check back later

**Comment by jinxynii** [comment]
I sent you the one I used, nevermind.

**Comment by jinxynii** [comment]
Do you have any intentions on giving this an auto aim clicker at all?

**Comment by SijyKijy** [comment]
> Do you have any intentions on giving this an auto aim clicker at all?

Nope

**Comment by jinxynii** [comment]
I've happened upon ByondControl's source code. Or atleast what appears to be it. Would you like me to send you it?

**Comment by SijyKijy** [comment]
> I've happened upon ByondControl's source code. Or atleast what appears to be it. Would you like me to send you it?

Hi! Yep

**Comment by jinxynii** [comment]
> > I've happened upon ByondControl's source code. Or atleast what appears to be it. Would you like me to send you it?
> 
> Hi! Yep

https://file.io/tgkfNFLGRf0z is the link. If you can get it to compile, do you mind sending me the stuff? For some reason Virtual Studio 2022 is being a massive pain in my ass and won't recognize installed SDKs.

**Comment by SijyKijy** [comment]
> do you mind sending me the stuff?

@jinxynii Do you mean compiled binaries?

**Comment by jinxynii** [comment]
> > do you mind sending me the stuff?
> 
> @jinxynii Do you mean compiled binaries?

Yeah. I can't get VS to recognize the fact that I have the required dependances installed ever since I updated to Windows 11, so they won't build.

**Comment by SijyKijy** [comment]
@jinxynii You can use Github Actions to build binary files as an option

**Comment by jinxynii** [comment]
I meant, I cannot build authlib, serialmath, or main because no matter what I do, ever since I updated to Windows 11, it will not recognize SDK installations as being installed. Even with proper paths defined. It wasn't an issue for your .dll, because it still worked then.

If you can get this to build, do you mind sending me the files afterwards, while I figure out *why* it won't recognize them?

**Comment by jinxynii** [comment]
> [@jinxynii](https://github.com/jinxynii) You can use Github Actions to build binary files as an option

Are you going to update this for 516?

**Comment by SijyKijy** [comment]
@jinxynii nope
